.github/workflows/integration-hydra.yml

```diff
@@ -58,29 +60,33 @@ jobs:
             ~/.cargo/registry/cache/
             ~/.cargo/git/db/
             ./rust/target
-          key: ${{ env.cache_id }}-${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}-${{ env.RUSTC_HASH }}
+          key:
+            ${{ env.cache_id }}-${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}-${{
+            env.RUSTC_HASH }}
 
       # Build Rust Program
       - uses: ./.github/actions/build-hydra
       - uses: ./.github/actions/build-token-metadata
 
       # Install JS SDK deps
       - uses: ./.github/actions/yarn-install-and-build
-        with: 
+        with:
           cache_id: sdk-hydra
           working_dir: ./hydra/js
           build_token_metadata: true
 
       # Run integration test
       - name: start-local-test-validator
         working-directory: ./hydra/js
-        run: yarn run api:gen && yarn run build && DEBUG=amman* yarn amman:start && DEBUG=mpl* yarn test && yarn run amman:stop
+        run:
+          yarn run api:gen && yarn run build && DEBUG=amman* yarn amman:start && DEBUG=mpl* yarn
```

Comment:
This reformatting looks odd as we got one command across multiple lines. Not sure how/if this could be improved.